ape_safe/accounts.py

```diff
@@ -85,7 +103,7 @@ def client(self) -> SafeClient:
     @property
     def version(self) -> str:
         try:
-            return self.client.safe_details.version
+            return self.client.safe_details.version.replace("+L2", "")
```

Comment:
Why do we want to remove this? Is an important component to the version imo.

Reply:
it's a fix for l2 safes, otherwise it breaks `eip712` library usage. plus below `VERSION()` is called, which would return `1.3.0` for both l1 and l2 versions of the contract.

Reply:
can probably add `.is_l2` property as well